contracts/test/mocks/L2LPTDataCacheMock.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+contract L2LPTDataCacheMock {
+    function l1CirculatingSupply() external pure returns (uint256) {
+        return 0;
```

Comment:
Since the minter contract is dependent upon this value for calculating inflation rate and subsequently rewards, I believe fixing this value to be 0 may cause some troubles if reward calls etc need to be tested.
(Though, It looks like these tests are not planned yet?) 

WDYT about creating a public variable which can be set accordingly via a task during tests and this function will simply return that value.

Reply:
> We use this mock contract only for the [geth-with-protocol](https://github.com/livepeer/docker-livepeer/tree/master/geth-with-protocol) docker image

gotcha 👍🏼 
In this case we can skip modifying the contract. PR good to go once Rafal's comments are addressed.